src/Notes/AddFirstProduct.php

```diff
@@ -63,7 +63,7 @@ public static function get_note() {
 			/* translators: %s: line break */
 			sprintf( __( 'Nice one; you\'ve created a WooCommerce store! Now it\'s time to add your first product and get ready to start selling.%s', 'woocommerce-admin' ), '<br/><br/>' ),
 			__( 'There are three ways to add your products: you can <strong>create products manually, import them at once via CSV file</strong>, or <strong>migrate them from another service</strong>.<br/><br/>', 'woocommerce-admin' ),
-			__( '<a href="https://docs.woocommerce.com/document/managing-products/?utm_source=help_panel">Explore our docs</a> for more information, or just get started!', 'woocommerce-admin' ),
+			__( '<a href="https://docs.woocommerce.com/document/managing-products/?utm_source=help_panel&utm_medium=product">Explore our docs</a> for more information, or just get started!', 'woocommerce-admin' ),
```

Comment:
I think we should use `sprintf` here to avoid breaking translations in the future.

Reply:
@joshuatf could you clarify how you want this to use `sprintf` do you want the `<a>` and `</a>` tags in variables?

Reply:
What I've usually done and seen in our repos is:

```php
sprintf( 'Here is a <a href="%s">link</a>', 'http://link.com' );
```

That said, I'm wondering if tossing the entire link into a variable would work so we could retain better control of the link (e.g., force using the `target` attribute).

Reply:
I just tested with:

```php
	sprintf(
		'Testing %1$slink%2$s',
		'<a href="http://google.com">',
		'</a>'
	)
```

And this does in fact work.  I can't see why we don't do it this way other than readability.  There are some plugins that seem to, but most seem to favor the method I used in the last comment.  @mattsherman would you by any chance have context on this?

Reply:
What you just presented is how I imagined we'd implement it. This should be the best way to separate markup from translation right?

Reply:
That's somewhat similar to how Calypso handles translations. It is still somewhat fragile (translators could mess up the order of the opening and closing tags), and it certainly isn't translator-friendly... we need to make sure we provide notes in the translators comments stating that `%1$s` and `%2$s` are opening and closing anchor tags. But I agree that this is the better approach.

Reply:
I've added comments for the translator, but I'll make it even more specific as suggested here.

Reply:
I've clarified the comment here: https://github.com/woocommerce/woocommerce-admin/pull/7408/commits/7d0a280562029882acede136f59b1f51dfacb31f